readme.md

```diff
@@ -57,4 +60,4 @@ This assumes that npm & typings & dojo-cli have been installed for the example a
 ./scripts/gh-pages-publish.sh
 ```
 
-The latest version(s) will now be available [here](https://dojo.github.io/examples).
\ No newline at end of file
+The latest version(s) will now be available [here](https://dojo.github.io/examples).
```

Comment:
I'm not a fan of links to here or this, because it reduces scannability... not that you did this... I would change it so the link test is something meaningful